doc/tutorial/index.rst

```diff
@@ -0,0 +1,216 @@
+.. _tutorial:
+
+===============
+Sphinx tutorial
+===============
+
+In this tutorial you will build a simple documentation project using Sphinx,
+and view it in your browser as HTML.
+The project will include narrative, handwritten documentation,
+as well as autogenerated API documentation.
+
+The tutorial is aimed towards Sphinx newcomers
+willing to learn the fundamentals of how projects are created and structured.
+You will create a fictional software library to generate random food recipes
+that will serve as a guide throughout the process,
+with the objective of properly documenting it.
+
+To showcase Sphinx capabilities for code documentation
+you will use Python,
+which also supports *automatic* documentation generation.
```

Comment:
In my understanding, texts should be folded by 80 chars instead of semantic line folding until the discussion, right? It seems some updates are needed.

Reply:
Sorry, a misunderstanding from my side. Fixing.

Reply:
Done!